samples/durable-client-managed-identity/functions-app/README.md

```diff
@@ -0,0 +1,33 @@
+# Azure Function App with Durable Function and Identity-Based Connection
+
+This project demonstrates an Azure Function App that invokes a Durable Function through a Durable Client using dependency injection and identity-based connection. Both the function host and the injected Durable Client are configured to use a storage connection with the default name `Storage`.
+
+## To make the sample run, you need to:
+
+1. Create an identity for your Function App in the Azure portal.
+
+2. Grant the following Role-Based Access Control (RBAC) permissions to the identity:
+    - Storage Queue Data Contributor
+    - Storage Blob Data Contributor
+    - Storage Table Data Contributor
+
+3. Link your storage account to your Function App by adding either of these two details to your `local.settings.json` file (for local development) or as environment variables in your Function App settings in Azure.
+    - AzureWebJobsStorage__accountName
+    - AzureWebJobsStorage__blobServiceUri, AzureWebJobsStorage__queueServiceUri and AzureWebJobsStorage__tableServiceUri
+
+4. Add the required identity information to your Functions App configuration.
+    - system-assigned identity: nothing needs to be provided.
+    - user-assigned identity: 
+      - AzureWebJobsStorage__credential: managedidentity
+      - AzureWebJobsStorage__clientId
+    - client secret application:
+      - AzureWebJobsStorage__clientId
+      - AzureWebJobsStorage__ClientSecret
+      - AzureWebJobsStorage__tenantId
+
+
+## Notes
+
+- The Azure Functions runtime requires a storage account to start, with the default connection name `Storage`.
+- For Durable Functions initiated by the Durable Client to run successfully, both the Durable Client and the Functions runtime must use the same storage account. If you need to use a custom storage connection name for the Durable Client, ensure you also update the Azure Functions runtime to use this custom name.
```

Comment:
hmm, I find it surprising that the DF Client and the runtime would need the same account. Why would that be the case? I thought DF could use a different account, always. Am I missing something? Would love @bachuv to chime in in case I'm missing something :)

Reply:
I tried the scenario where df client and the runtime use different storage account. The client could create orchestration instances, but all the instances will remain in Pending state. So to make those orchestrators to start/ go to Running state, we need a runtime at the same storage account

Reply:
Ah, I see. I think that's expected though :) . If the storage account is different, then the expecation is that there is **another app** that is listening to that account that will then process those orchestrators.

In other words, in this scenario you have two apps: A and B.

App A's Functions Host and DF uses storageA
App B's Functions Host and DF uses storageB

Now, app A is allowed to create a DurableClient that references storageB and schedules an orchestrator _there_. That will cause app B to pick up that work and process it. This is what we call an "external client", because app A's client is external to the storage account where it is creating an orchestrator. Does that make sense?

Reply:
Thanks! That makes sense to me. I will update the notes.